README.md

```diff
@@ -131,14 +131,20 @@ $ kubeless function call get-python --data '{"echo": "echo echo"}'
 {"echo": "echo echo"}
 ```
 
-Or you can curl it directly if you run `kubectl proxy` first. For example:
+Or you can curl directly with `kubectl proxy`
+using an [apiserver proxy
+URL](https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/#manually-constructing-apiserver-proxy-urls).
+For example:
 
 ```console
 $ kubectl proxy -p 8080 &
 
-$ curl --data '{"Another": "Echo"}' localhost:8080/api/v1/proxy/namespaces/default/services/get-python/ --header "Content-Type:application/json"
+$ curl -L --data '{"Another": "Echo"}' localhost:8080/api/v1/proxy/namespaces/default/services/get-python:function-port/ --header "Content-Type:application/json"
 {"Another": "Echo"}
 ```
+> Note: You must use the port name in the url. To
+> find the port name, inspect the service using
+> `kubectl get svc get-python -o yaml`
```

Comment:
This should always be true, due to https://github.com/kubeless/kubeless/blob/bbd8c1deecf53a49bc52da4c20e61dc1d1788bb2/pkg/utils/k8sutil.go#L871 introduced recently in https://github.com/kubeless/kubeless/commit/0c7e69cbae588f142db8b3f94fb2fa52f12cd42e and released as part of v0.2.3

So: I think we can remove this extra step, and just have the above URL use `:function-port` without further explanation.

Reply:
I agree